java-desktop/src/main/java/com/couchbase/todo/controller/ShareController.java

```diff
@@ -87,16 +79,15 @@ private void getUserList() {
 
         DB.get().addChangeListener(query, change -> {
             ObservableList<User> users = FXCollections.observableArrayList();
+            assert change.getResults() != null;
```

Comment:
Interesting.  Won't this get an NPE on the next line, if change.getResults is null?
If you think this adds value, I'm totally good with it.  Have a look, though, at `getResults` to see how expensive it is to call it twice...

Reply:
so what should happen if change.getResults() returns null? because right now the warning said it could be null